frontend/src/features/properties/hooks/useGeocoder.tsx

```diff
@@ -116,14 +117,28 @@ const useGeocoder = ({ formikRef, fetchPimsOrLayerParcel }: IUseGeocoderProps) =
             handleParcelDataLayerResponse(response, dispatch);
           };
           fetchPimsOrLayerParcel &&
-            fetchPimsOrLayerParcel(parcelPid, parcelLayerSearchCallback, nameSpace);
+            fetchPimsOrLayerParcel({ pid: parcelPid }, parcelLayerSearchCallback, nameSpace);
         } else {
           if (data.latitude && data.longitude) {
-            const parcelDataLayerResponse = parcelsService.findOneWhereContains({
-              lat: data.latitude,
-              lng: data.longitude,
-            } as LatLng);
-            handleParcelDataLayerResponse(parcelDataLayerResponse, dispatch);
+            parcelsService
+              .findOneWhereContains({
+                lat: data.latitude,
+                lng: data.longitude,
+              } as LatLng)
+              .then(response => {
+                const pid = getIn(response, 'features.0.properties.PID');
+                //it is possible the the geocoder will fail to get the pid the parcel layer service will succeed. In that case, double check that the pid doesn't exist within pims.
```

Comment:
I'm having a hard time reading this sentence, are some words missing?

Reply:
will fix.